### ISSUE TITLE
Last game option selected still highlighted after quit (Tutorial)

### ISSUE THREAD

**Issue by tjdunlap** [roniRamon/big-project#55]
Quit the game to restart, and the option I choose in the last round was already highlighted. It's kind of interesting, to be honest. :)

Attempted to replicate: At first, there was no luck, but It seems the options once selected will become highlighted, and remain as such once restarting a new game in the same page.
Sometimes the options will bounce to another, and sometimes they will not change.

(Tutorial) 
-Fight the Enemy          } Bounces between options based on last selected
-Run from the enemy  ^
-Keep the item                           } Bounces between options based on last selected
-Ignore the Item and move on ^
(Forest) First Page options highlight will bounce and remain on last selected in the next session after quitting
(Mountain) First Page options highlight will bounce and remain on last selected in the next session after quitting
(Desert) First Page options highlight will bounce and remain on last selected in the next session after quitting
(Python's Nest) - Both options remain highlighted into next session

**Comment by roniRamon** [comment]
is that causing an issue with the game?

**Comment by tjdunlap** [comment]
Not really. Just thought I'd mention it as I'm not sure if it was intentional. In fact, I think it's kind of nice to have about as it could remind players of what action they chose last round. :)
